tools: Use 'make-rpms' to build host native packages

stefwalter opened the pull request:
This is the quick path to testing out a Cockpit commit on an RPM
based system. It doesn't require mock, or a VM or anything else
besides all the development dependencies.

mvollmer commented on tools/make-rpms:
```diff
@@ -16,39 +16,23 @@
-	-q|--quick)
-		mock_define="selinux" mock_value="0"
-		mock_opts="$mock_opts --offline --nocheck"
+        opts=""
```
Indentation

stefwalter replied:
Fixed.

mvollmer commented on tools/make-rpms:
```diff
@@ -16,39 +16,23 @@
+opts="--quiet"
 
-args=$(getopt -o "h,v,c,q,r:" -l "help,verbose,clean,quick" -- "$@")
+args=$(getopt -o "h,v,q,r:" -l "help,verbose,quick" -- "$@")
```
--quick is not implemented.  I think we should.

stefwalter replied:
Removed.

dperpeet commented on tools/make-rpms:
```diff
@@ -67,82 +51,17 @@ if [ $# -ne 0 ]; then
+    --define "_builddir `pwd`" \
+    --define "_srcrpmdir `pwd`" \
+    --define "_rpmdir $tmpdir" \
+    --define "_buildrootdir `pwd`/.build" \
```
this doesn't get deleted after the build

stefwalter commented:
Changed to do all building inside of a temporary directory, which gets removed afterwards. Also added back the ```--quick``` option, which skips checks.

mvollmer commented on tools/cockpit.spec:
```diff
@@ -142,13 +142,15 @@ The Cockpit Web Service listens on the network, and authenticates users.
 env NOCONFIGURE=1 ./autogen.sh
 %endif
-%configure --disable-static --disable-silent-rules --with-cockpit-user=cockpit-ws --with-branding=%{branding} --with-selinux-config-type=etc_t
+%configure --disable-silent-rules --with-cockpit-user=cockpit-ws --with-branding=%{branding} --with-selinux-config-type=etc_t
```
This looks like an unrelated change. Why is --disable-static removed?

stefwalter replied:
Because it's not supported, and just makes configure.ac spew noise. The commit this is in is one that removes noise.

mvollmer replied:
Thanks, makes sense.
